src/commands/auth/store-token/store.ts

```diff
@@ -0,0 +1,76 @@
+import path from 'path';
+import os from 'os';
+import fs from 'fs/promises';
+import keytar from 'keytar';
+import { decrypt, encrypt } from './encrypt.js';
+import { getUser } from '../api/user.js';
+
+// Constants for the application
+export const SERVICE_NAME = "AnyFlowCLI";
+export const ACCOUNT_NAME = "apiToken";
+
+// Store the token securely
+export async function storeToken(token: string) {
+  try {
+    // Verify the token by getting user info
+    await getUser(token);
+
+    // Attempt to store the token using the system keychain
+    await keytar.setPassword(SERVICE_NAME, ACCOUNT_NAME, token);
+
+    console.log("Token and user info stored securely using system keychain.");
+  } catch (error) {
+    console.error("Error storing token:", error);
```

Comment:
This prints out the whole error when we input an invalid token. This is not good UX. You should handle the invalid token errors and provide a short feedback to the user.

![image](https://github.com/user-attachments/assets/a66fd9aa-5968-4b45-bc44-3441a4b0ebca)